dedupe.nf

```diff
@@ -0,0 +1,18 @@
+// Dedupe - deduplication of sequencing reads
+// Laura Holland lholland@arkeabio.com
+// Arkea Bio Corp, May 2023
+
+include { BBMAP_DEDUPE } from './modules/nf-core/bbmap/dedupe/'
+
+read_ch = Channel.fromFilePairs(params.pairedreads, checkIfExists: true)
+
+read_ch.view()
```

Comment:
This isn't strictly necessary for modules, I like to keep it in to make sure my channel is loading things correctly. We'll want to remove most of these for any full releases of the pipeline.